Formula/pet.rb

```diff
@@ -0,0 +1,61 @@
+class Pet < Formula
+  desc "Polyhedral Extraction Tool"
+  homepage "https://pet.gforge.inria.fr/"
+  url "http://pet.gforge.inria.fr/pet-0.11.5.tar.xz"
+  sha256 "e9238a3e92cc13358cdec22d94c23740ac758f649df891f4bc3461e988e879e3"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+    regex(/pet-(\d+(?:\.\d+)+b?)/i)
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "gmp"
+  depends_on "isl"
+  depends_on "libyaml"
+  depends_on "llvm"
+
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  def install
+    site_packages = Language::Python.site_packages(Formula["python@3.9"])
+
+    # avoid using llvm's libc++
+    ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
+    system "./configure", "--prefix=#{prefix}", "--with-isl=system"
+    system "make"
+    system "make", "install"
+
+    (lib/"pkgconfig").install "pet.pc"
+    (prefix/site_packages).install "interface/pet.py"
```

Comment:
Isn't `make install` supposed to install this? Can we report upstream that it isn't being installed? (Or is not installing this intentional?)

Reply:
Auto generated file need to be installed manually.

Reply:
Yes, but other build systems have auto-generated files that do not need to be installed manually. (Note that `bin/"pet"` is also auto-generated, but we don't install that manually.) Why does their build system not install these with `make install`?